ui/packages/catalog-ui-search/src/main/webapp/component/search-form/search-form.less

```diff
@@ -20,14 +21,20 @@
     }
 
     .search-form-label {
-        .is-bold();
+        font-weight: bold;
     }
+
+    .choice-actions {
+        position: absolute;
+        right: 10px;
+        bottom: 0px;
+      }
 }
 
 @{customElementNamespace}search-form.is-static {
     text-align: center;
 }
 
 @{customElementNamespace}search-form-loading {
-    padding: 2 * @minimumSpacing;
+    padding: 2*@minimumSpacing;
```

Comment:
nit: Add spacing to all mathematical operators

Reply:
fixed